modules/core/src/shaderlib/project/project.js

```diff
@@ -36,30 +36,5 @@ export default {
   dependencies: [fp32],
   vs: projectShader,
   getUniforms,
-  deprecations: [
-    // Removed custom picking uinforms
-    // These don't really belong here but we need to check them for all shaders
-    // the project module is by default included for all
-    {type: 'uniform vec3', old: 'selectedPickingColor', new: "luma.gl's picking module"},
```

Comment:
Given some apps are still on v5 my sense would be to cut v4->v5 warnings but not v5->v7 as these are invaluable when upgrading custom layers.

Mark with comments which release each block of warnings belong to make staggered removals easier in the future?

Reply:
We have already removed v5->v6 deprecation warnings from the rest of this repo. I do not think we should keep them indefinitely just because some users have not upgraded. The number of v5 custom layers in the wild is pretty low...